tests/test_tw_surface.rs

```diff
@@ -152,3 +152,25 @@ fn create_stylesheet() -> CommonResult<Stylesheet> {
     stylesheet
   })
 }
+
+/// Create a stylesheet containing styles using DSL.
+fn dsl_stylesheet() -> CommonResult<Stylesheet> {
+  throws_with_return!({
+    stylesheet! {
```

Comment:
The indentation in this macro & in other macros is different from `rustfmt.toml` here https://github.com/r3bl-org/r3bl-rs-utils/blob/Kofituo-req/rustfmt.toml